GETTING STARTED - Developers.rdoc

```diff
@@ -0,0 +1,61 @@
+==Is this doc for you?
+
```

Comment:
I'm not sure this is the right name, but I think we need an introduction point for developers in the code, and I don't think README is right.